torch/onnx/utils.py

```diff
@@ -229,6 +269,11 @@ def _model_to_graph(model, args, f, verbose=False, training=False,
         graph, torch_out = _trace_and_get_graph_from_model(model, args, training)
         params = list(_unique_state_dict(model).values())
 
+        all_names = list(_unique_state_dict(model).keys())
```

Comment:
Please note that currently this does not handle the ScriptModule code path above in `_model_to_graph()`. Primarily because I haven't been able to successfully export an (simple) LSTM node using that code path. If I can get some help on that I can fix that code path too. Also, related to that, is there a way to obtain the parameter names in that code path (maybe through `method` object)?

Reply:
@spandantiwari could you elaborate more on the problem description? So I can reproduce problem locally, thanks